src/navbar/navbar-items.styles.tsx

```diff
@@ -110,19 +106,17 @@ export const LinkIconContainer = styled.div`
     padding-left: 1rem;
 `;
 
-export const DownIcon = styled(ChevronDownIcon)`
-    height: 1.25rem;
-    width: 1.25rem;
-
-    :hover {
-        color: ${Color.Accent.Light[1]};
-    }
+export const ExpandCollapseButton = styled(IconButton)<StyleProps>`
+    padding: 0.5rem;
+    transform: rotate(${(props) => (props.$selected ? 0 : 180)}deg);
+    transition: transform 300ms ease-in-out;
+    margin: auto 0.25rem auto 0;
 `;
 
-export const UpIcon = styled(ChevronUpIcon)`
-    height: 1.25rem;
-    width: 1.25rem;
-
+export const ChevronIcon = styled(ChevronUpIcon)`
+    height: 1rem;
+    width: 1rem;
```

Comment:
this should be 1.25rem based on figma

the padding to the right edge should also be 1.375rem